src/daqlite/CustomTofPlot.cpp

```diff
@@ -112,3 +118,17 @@ void CustomTofPlot::clearDetectorImage() {
   addData(HistogramTofData);
   plotDetectorImage(true);
 }
+
+// MouseOver, display coordinate and data in tooltip
+void CustomTofPlot::showPointToolTip(QMouseEvent *event) {
+  int x = this->xAxis->pixelToCoord(event->pos().x());
+
+  // Calculate coordinates grouped with bins and retrive data
+  // according to the identified bin.
+  int stepSize = int(mConfig.TOF.MaxValue / mConfig.TOF.BinSize);
+  int dataBinPos = int((x - stepSize / 2) / stepSize);
```

Comment:
so this is actually:
x/stepsize -1/2  right? Took me a while to figure out. Maybe change the formula a little?

Reply:
And maybe add a little more comments here. I assume there is some reason behind the -1/2 but I don't know it.

Reply:
So you create a bin to group those tof which are similar and then you load these to the dataset so there will be 512 values. However the QT chart representation will draw them according to the TOF max value and create MaxValue/BinSize wide columns. But when we move the mouse over the chart we get the absolute x/y coordinate according to the axis not according to the drawn columns.